site/in-dev/configuration.md

```diff
@@ -68,6 +68,7 @@ Related Quarkus settings:
 | MongoDB          | production                                       | `MONGODB`                                               |                                                                                                                                                                                                                                 |
 | Amazon DynamoDB  | beta, only tested against the simulator          | `DYNAMODB`                                              |                                                                                                                                                                                                                                 |
 | PostgreSQL       | production                                       | `JDBC`                                                  |                                                                                                                                                                                                                                 |
+| MariaDB          | experimental, feedback welcome                   | `JDBC`                                                  | This backend can also be used with a MySQL server.                                                                                                                                                                              |
```

Comment:
Should we mention MySQL here as well. Mean - mentioning is not distributing. WDYT?

Reply:
Yep you are right. I was being overly cautious.